tools/pd-ut/ut.go

```diff
@@ -118,10 +118,17 @@ func main() {
 		defer os.RemoveAll(coverFileTempDir)
 	}
 
-	// Get the correct count of CPU if it's in docker.
-	p = runtime.GOMAXPROCS(0)
-	// We use 2 * p for `go build` to make it faster.
-	buildParallel = p * 2
+	if parallelStr == "" {
+		// Get the correct count of CPU if it's in docker.
+		parallel = runtime.GOMAXPROCS(0)
+	} else {
+		var err error
+		parallel, err = strconv.Atoi(parallelStr)
```

Comment:
Do we need check if the `parallel<=runtime.GOMAXPROCS(0)`

Reply:
There is no need, we can support parallel greater than procs. It depends on user

Reply:
make sense, and add a hint when parallel was set greater than procs :)
https://github.com/tikv/pd/pull/8186/commits/f42fc5e450d1eaa05d21d9b7cefbd291712dabd1#diff-d1de7e6511458d33a769c0c501ec0ea45494dd9d2e830dc315e8fe623eb34b76R132